.vscode/launch.json

```diff
@@ -6,13 +6,9 @@
             "type": "extensionHost",
             "request": "launch",
             "runtimeExecutable": "${execPath}",
-            "args": [
-                "--extensionDevelopmentPath=${workspaceRoot}"
-            ],
+            "args": ["--extensionDevelopmentPath=${workspaceRoot}"],
```

Comment:
Some of these changes came from the prettier json formatter. Idk if it matters, I just noticed some things being spaced differently and ran a formatter